src/constructors.jl

```diff
@@ -34,6 +34,14 @@ end
 
 @compat (::Type{NullableArray{T}}){T}(dims::Dims) = NullableArray(T, dims)
 @compat (::Type{NullableArray{T}}){T}(dims::Int...) = NullableArray(T, dims)
```

Comment:
Ah, right, I thought `N` could be inferred automatically in your new constructor, but that doesn't work for the call overloading syntax.